swiftide/src/integrations/aws_bedrock/simple_prompt.rs

```diff
@@ -53,7 +53,7 @@ mod test {
             .build()
             .unwrap();
 
-        let response = bedrock.prompt("Hello").await.unwrap();
+        let response = bedrock.prompt(&"Hello".into()).await.unwrap();
```

Comment:
There has to be a nice way to not have this breaking change and just allow &'static str. `impl` does not work with automock and trait bounds in traits are meh.